habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1376,6 +1417,68 @@ def _apply_velocity_action(
 
         return final_agent_state
 
+    def _apply_camera_pitch_velocity_action(
+        self,
+        camera_pitch_angular_velocity: float,
+        time_step: Optional[float] = None,
+    ):
+        """
+        Apply velocity command to the camera tilt angle for looking up and down
+        """
+        if time_step is None:
+            time_step = self._time_step
+
+        # Map velocity actions
+        self.vel_control.linear_velocity = np.array([0.0, 0.0, 0.0])
```

Comment:
Can you make them mn.Vector3 similar to what I did for `_apply_velocity_action` function

Reply:
Thank you!